<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>EzCross</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="Line" name="split_lr">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>101</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_host">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_host">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_username">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_password">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>341</width>
     <height>105</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBox_remerber_me">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Remerber Me</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="Button_connect">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Port">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_port">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QComboBox" name="comboBox_user_choice">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Default</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_login_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>631</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_local_browser">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Browser</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_local">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>/mnt/h/code_repo/project</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_local">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Local</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_remote">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Remote</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_remote">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>/home/user/source/project</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_remote_browser">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Browser</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>631</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_build_tool">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Build Tool</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_build_tool">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Makefile</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CMake</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ninja</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>...</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>631</width>
     <height>119</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_build_command">
      <property name="text">
       <string>Build Command</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_build_command"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>631</width>
     <height>119</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_run_command">
      <property name="text">
       <string>Run Command</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_run_command"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QCheckBox" name="checkBox_run_it">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Run it after built</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_download_it">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Download the binart file to local after built</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>510</y>
     <width>631</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>550</y>
     <width>271</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_build">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Build</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_upload">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit_showback">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
